SPEC.md

```diff
@@ -5081,9 +5117,11 @@ Test config:
     * `Array[String]` - An array of disk specifications.
 * Default value: `1 GiB`
 
-The `disks` attribute provides a way to request one or more persistent volumes, each of which has a minimum size and is mounted at a specific location. When the `disks` attribute is provided, the execution engine must guarantee the requested resources are available or immediately fail the task prior to instantiating the command.
+The `disks` attribute provides a way to request one or more persistent volumes, each of which has a minimum size and is mounted at a specific location with both read and write permissions. When the `disks` attribute is provided, the execution engine must guarantee the requested resources are available or immediately fail the task prior to instantiating the command.
+
+If the mount point is omitted, it is assumed to be a persistent volume mounted at the root of the execution directory within a task.
 
-If a mount point is specified, then it must be an absolute path to a location in the host environment. If the mount point is omitted, it is assumed to be a persistent volume mounted at the root of the execution directory within a task.
+If a mount point is specified, then it must be an absolute path to a location in the host environment (i.e., within the container). The specified path either must not already exist in the host environment, or it must be empty and have at least the requested amount of space available. The mount point should be assumed to be ephemeral, i.e., it will be deleted after the task completes.
```

Comment:
I've always seen "host environment" in the context of containerized applications used to mean the system *outside* the container; not the system *within* the container. If I am on a machine booted into Fedora and I run a WDL workflow that calls for an `ubuntu:22.04` container to run, the "host" as I understand it would be the Fedora environment.

Also, I don't think that containers have a notion of some amount of space being available at a path in the container. I don't think I can make an image that has 20 GiB of free space at `/mnt/freespace` and upload it to Docker Hub, as far as I know. So I don't think saying `must be empty and have at least the requested amount of space available` is different from just saying `must be empty`. You can mount at paths that don't exist in your container image, or you can mount at paths that exist as empty directories, but you can't mount at paths that are files or paths that are directories that have contents.

Reply:
Yes, we should be more consistent with usage. I've updated the spec such that all uses of "execution environment" refers to the context in which the task is executed, and "host" refers to the system that hosts the execution environment.